image/histogram_s4443048_Puyang_Deng/histogram.py

```diff
@@ -0,0 +1,158 @@
+import tensorflow as tf
```

Comment:
Always good to add a header block. Block would include authorship, license, copyright, dates and description of what the driver/module does.

Reply:
Added the header. Sorry for the delay!